<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-5FCAB7O" ignored="0" name="BankingApp Test" tests="7" failures="2" timestamp="04 Sep 2019 07:06:50 GMT" time="75.39" errors="0">
  <testcase name="loginTest" time="2.381" classname="com.inetbanking.testCases.TC_LoginTest_001"/>
  <testcase name="loginDDT" time="8.767" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="4.778" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:32)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="8.233" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="4.332" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:32)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="8.366" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="addNewCustomer" time="15.706" classname="com.inetbanking.testCases.TC_AddCustomerTest_003"/>
</testsuite> <!-- BankingApp Test -->
